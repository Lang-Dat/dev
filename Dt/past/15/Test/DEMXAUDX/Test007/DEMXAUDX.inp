161 142
ADSASNNDANDADMMDASASADMAAMSDMNNAASMNDMMMDNMSNSNMASDDSNSNDASASNDANSANMMSSMDSSDAADAMSDMDNMDDDSASSNNDSMMDDNDSNAADDAASSDNMSMDNSDNSNMASSAMNSMSNSSNSMSASMMMSNAMMSSSANND
29 59
43 117
13 116
130 141
123 146
9 134
86 157
7 118
14 133
2 155
148 154
82 155
7 143
8 131
30 89
8 145
39 128
13 148
4 160
107 122
4 128
74 79
115 156
9 142
98 137
11 147
20 48
4 132
42 134
136 144
91 136
13 139
53 159
143 144
94 144
10 157
8 119
155 156
6 137
13 119
61 81
143 147
4 135
63 114
5 143
76 79
18 72
11 114
16 148
141 156
15 135
67 160
1 119
9 126
58 86
3 6
50 106
127 143
113 143
8 131
7 111
35 143
1 147
4 160
113 153
13 152
14 120
4 137
3 112
116 122
61 78
144 144
144 156
89 130
8 80
99 134
12 128
102 149
19 74
12 134
10 127
6 126
93 112
18 111
11 118
89 148
2 133
5 152
9 159
25 70
16 111
138 145
19 106
12 114
15 122
9 129
9 116
14 127
13 155
14 143
86 148
13 119
6 148
2 139
9 134
25 88
3 129
121 154
92 143
14 138
16 136
12 136
107 134
10 153
11 112
6 137
27 56
135 141
57 58
100 110
97 115
12 41
1 126
2 135
13 114
13 135
13 156
71 83
21 72
13 150
16 136
12 131
12 152
147 148
14 156
1 26
88 126
8 151
158 161
9 148
16 135
12 127
