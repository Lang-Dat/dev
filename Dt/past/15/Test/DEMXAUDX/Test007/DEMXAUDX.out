46
107
155
18
41
185
114
165
175
233
9
114
205
180
86
208
125
207
239
23
181
9
67
201
57
209
42
187
131
16
65
188
162
2
79
228
165
3
194
158
29
7
193
75
208
5
78
156
201
25
177
146
174
172
40
5
81
29
47
180
154
162
221
239
65
214
158
196
162
8
23
1
20
59
109
51
171
78
79
181
173
176
28
135
161
95
192
225
231
64
140
12
126
154
158
176
161
166
218
195
98
158
217
203
185
90
184
57
79
184
176
184
40
220
152
194
45
11
2
15
28
47
182
196
152
181
220
19
73
212
176
175
216
3
218
40
55
220
5
213
175
170
